_scripts/clone-legal-docs.sh

```diff
@@ -130,25 +134,33 @@ copy_md() {
 
     log "Copying $md_name and parent locale directories into $MODULE_PATH/public/legal-docs/"
 
+    cd "$DOWNLOAD_PATH"
     results=()
     for src in **"/${md_name}.md"; do
         [ -f "$src" ] || continue
-        legal_docs_dir="$MODULE_PATH/public/legal-docs/$(dirname "$src" | sed -E 's/^.*\/([a-zA-Z_]+)\/.+/\1/; s/_/-/g')"
+        legal_docs_dir="$LOCAL_PATH/$(dirname "$src" | sed -E 's/^.*\/([a-zA-Z_]+)\/.+/\1/; s/_/-/g')"
         [ -d "$legal_docs_dir" ] || mkdir -p "$legal_docs_dir"
         cp "$src" "$legal_docs_dir/$md_name.md"
         results+=("$(echo "$legal_docs_dir" | sed -E 's/.+\/([a-zA-Z_]+)/\1/; s/-/_/g')")
     done
 
-    log "Creating .json file containing array of available locales in $MODULE_PATH/public/legal-docs/${md_name}_locales.json"
+    # log "Creating .json file containing array of available locales in $MODULE_PATH/public/legal-docs/${md_name}_locales.json"
```

Comment:
Do you mind updating the comment at the top of this file to reflect this change? 🙏